projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/my-company/config/basesites.config.ts

```diff
@@ -0,0 +1,312 @@
+export const baseSitesConfig = {
```

Comment:
why we put this config inside "my-company"? I don't think we need it. We can just create a mock baseSiteConfig in b2b-secure-portal.ts, which only contains one site with properties `uid`, `requiresAuthentication: true,`